File: C_Programming/Competitive_Programming/CP_004.c
////////////////////////////////////////////////////////////////////
//
//  File Name    : Check_Divisibility_By_Five
//
//  Description  : This program accepts a number from the user
//                 and checks whether it is divisible by 5.
//
//  Example:
//      Input  : 10
//      Output : Divisible by 5
//
//      Input  : 18
//      Output : Not Divisible by 5
//
////////////////////////////////////////////////////////////////////

#include<stdio.h>

typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL Check(int iNo)
{
    if((iNo % 5 == 0))
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter number : \n");
    scanf("%d",&iValue);

    bRet = Check(iValue);

    if(bRet == TRUE)
    {
        printf("Divisible by 5");
    }
    else
    {
        printf("Not Divisible by 5");
    }

    return 0;
}

File: Java_Programming/Competitive_Programming/CP_004.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : Find_Minimum_Of_Three_Numbers
//
//  Description  : This program accepts three numbers and
//                 displays the smallest number.
//
//  Example:
//      Input  : 3, 7, 2
//      Output : Smallest Number is : 2
//
//////////////////////////////////////////////////////////////////

class Logic
{
    void findMin(int a, int b, int c)
    {
        if((a < b) && (a < c))
        {
            System.out.print("Smallest Number is : " + a);
        }
        else if((b < a) && (b < c))
        {
            System.out.print("Smallest Number is : " + b);
        }
        else if((c < a) && (c < b))
        {
            System.out.print("Smallest Number is : " + c);
        }
    }
}

class CP_004
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();

        lobj.findMin(3,7,2);
    }
}